package.json

```diff
@@ -26,7 +26,7 @@
     "@nestjs/platform-fastify": "^10.4.6",
     "@nestjs/swagger": "^8.0.2",
     "@prisma/client": "^5.22.0",
-    "axios": "^1.7.7",
```

Comment:
why did you remove axios?
https://docs.nestjs.com/techniques/http-module
the docs instruct to install both nextjs/axios and axios

Reply:
I noticed we weren't improving it anywhere and so I removed it and your endpoints still operate as expected? 🤷‍♂️

Pull my branch and see if you get some kind of error maybe? Worked for me.

Reply:
Maybe they want it installed because they only have it as a devDependency? Maybe it won't work with a production build? 🤔

https://github.com/nestjs/axios/blob/master/package.json#L32